### ISSUE TITLE
Request: Allow for PSSDiag customer-side scheduling programmability (masha) 

### ISSUE THREAD

**Issue by MashaMSFT** [microsoft/SqlNexus#58]
I spend a lot of time with customers trying to determine which parameters I should pass when starting the PSSDiag in order to successfully collect data at a certain time, or under certain conditions. 

It would  be AMAZING if this could be done client side by the customer! Maybe launching PSSDiag.exe could actually launch a little  customizable GUI programlet thingy that lets the customer pick the parameters they want to use for running the PSSDiag?

For example, if the customer wants to create a new output folder every N number of hours, they can check the box for that, fill in how many hours they want and the parameters would automatically be passed to do so when launching the actual collection?

Another great option would be to schedule the PSSDiag to run as a service at a particular time.. Like in the program we can specify that the program can run when the session is logged out, but run from 1am to 2am every night.

So, in summary, if we were able to create this GUI interface, it should allow customers to select options for the following
- Recycling output (should a new folder be created? should a new folder over write the old one? 
- Scheduling the PSSDiag at a certain time, and maybe repeating daily
- Configuring the PSSDiag to run as a service (and if so, which account should be used) 
- Maybe deciding how many trace files to grab, and letting them choose how large (with a drop down of like 256mb, or 512mb, we don't want to give them too much power!) 

I don't even know if such a thing is possible, but it would be such a HUGE GAME CHANGER to our support staff if the customer had this ability built in. 

Thank you!!!

**Comment by PiJoCoder** [comment]
Better late than never. Closing this as moving it to the right repo for the right product. 
BTW, you can schedule PSSDIAG to run as as service at a particular time. 

[/B [+]YYYYMMDD_HH:MM:SS] = specifies the date/time to begin collecting data; "+" specifies a relative time
[/R]  = registers the collector as a service

For example /B 11:58:00 /R
